src/Sylius/Behat/Context/Ui/Admin/ManagingPromotionsContext.php

```diff
@@ -617,6 +620,26 @@ public function thePromotionsShouldHavePriority(PromotionInterface $promotion, i
         Assert::same($this->updatePage->getPriority(), $priority);
     }
 
+    /**
+     * @Then the :promotion promotion should have :ruleName rule configured
+     */
+    public function thePromotionShouldHaveRuleConfigured(PromotionInterface $promotion, string $ruleName): void
+    {
+        $this->iWantToModifyAPromotion($promotion);
```

Comment:
Can we also make a `submit` on this page? Just to ensure, that clicking a button will not 💥

Reply:
Added in other step